include/modules/script.hpp

```diff
@@ -24,16 +26,20 @@ namespace modules {
     bool check_condition();
 
    private:
-    static constexpr const char* TAG_LABEL{"<label>"};
+    static constexpr auto TAG_LABEL = "<label>";
+    static constexpr auto TAG_LABEL_FAIL = "<label-fail>";
+    static constexpr auto FORMAT_FAIL = "format-fail";
 
     const bool m_tail;
     const script_runner::interval m_interval{0};
+    const bool m_has_format_fail;
```

Comment:
This is not needed, you can use `m_formatter->has_format` to check if the fail format exists.